contrib/NeonVM/neonvm.c

```diff
@@ -344,39 +413,48 @@ void exec_LOADB()
     fatal_error("not implemented");
 }
 
-void exec_LOADN()
+void exec_LOADN(struct tagTExecutor *self)
 {
-    fatal_error("not implemented");
+    self->ip += 1;
+    Cell *addr = top(self->stack)->address; pop(self->stack);
+    push(self->stack, addr);
 }
 
-void exec_LOADS()
+void exec_LOADS(struct tagTExecutor *self)
 {
+    *self = *self;

```

Comment:
What exactly is this intended to accomplish? If you're trying to suppress an "uninitialized variable" warning, just 'self = self;' is sufficient. You don't need to copy the whole structure to itself.